Microsoft.Toolkit.Uwp.Input.GazeControls/GazeFilePicker.xaml.cs

```diff
@@ -0,0 +1,390 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.Input.GazeInteraction;
+using Windows.Storage;
+using Windows.Storage.FileProperties;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Microsoft.Toolkit.Uwp.Input.GazeControls
+{
+    /// <summary>
+    /// Provides file picker dialogs optimized for gaze input
+    /// </summary>
+    public sealed partial class GazeFilePicker : ContentDialog, INotifyPropertyChanged
```

Comment:
We shouldn't use `INotifyPropertyChanged` for controls, we should use Dependency Properties.

Reply:
Updated with Dependency properties